src/poliastro/tests/tests_twobody/test_propagation.py

```diff
@@ -10,12 +11,13 @@
 from poliastro.constants import J2000
 from poliastro.bodies import Sun, Earth
 from poliastro.twobody import Orbit
-from poliastro.twobody.propagation import cowell
+from poliastro.twobody.propagation import cowell, kepler, mean_motion
```

Comment:
Also, can you add a test that demonstrates that the new `mean_motion` propagator works with the Halley orbit, and another one marked as `@pytest.mark.xfail` with the `kepler` propagator?